### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by PowerPress** [TweekFawkes/White_Lightning#8]
This is a cool project, is it still being developed or is it abandoned?

**Comment by Leowulf** [comment]
I agree it is very cool, but it has not received much attention from the author lately. I contacted him about 3 weeks ago about testing the software on a localhost and he kindly responded, however, further inquiries were left without replies. 

How about we figure this out together on our own? Have you done any tinkering with it?
